src/ui/tools/filterutils.js

```diff
@@ -0,0 +1,21 @@
+import FilterCombinators from '../../core/filters/filtercombinators';
+import isEqual from 'lodash.isequal';
+import Filter from '../../core/models/filter';
+
+/**
+ * Checks whether the given filter is included or equal to
+ * the filter persisted in the URL. Assumes the given filter
+ * is a simple filter, i.e. does not have any child filters.
+ *
+ * @param {Filter} filter
+ * @param {Filter} persistedFilter
+ * @returns {boolean}
+ */
+export function filterIsPersisted (filter, persistedFilter) {
+  const childFilters =
+    persistedFilter[FilterCombinators.AND] || persistedFilter[FilterCombinators.OR];
+  if (childFilters) {
+    return !!childFilters.find(childFilter => filterIsPersisted(filter, Filter.from(childFilter)));
```

Comment:
If we're assuming there are no child filters (re: jsdoc), should we be doing this recursion?

Reply:
the jsdoc is for the filter to be tested not the persisted filter, I can update the jsdoc